units/csv_log_tests.cc

```diff
@@ -32,8 +39,8 @@ TEST(CSVLogger, MethodCheck) {
   logger.EndLog();
   // Checks double `EndLog` calls.
   logger.EndLog();
-
-  SUCCEED();
+  EXPECT_THAT(out.GetLog(), ElementsAre("pipeline,duration",
+                                        EventToCSVString(pipeline_event)));
```

Comment:
Can you check for the exact expected pipeline event? As-is, if `EventToCSVString` breaks, we won't notice any issues